src/utils/SignalingSpeakingHandler.js

```diff
@@ -0,0 +1,127 @@
+/**
+ *
+ * @copyright Copyright (c) 2023 Maksim Sukharev <antreesy.web@gmail.com>
+ *
+ * @author Maksim Sukharev <antreesy.web@gmail.com>
+ *
+ * @license AGPL-3.0-or-later
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+/**
+ * Helper to handle signaling messages for speaking / speakingStopped notifications.
+ *
+ * The store is updated when local or remote participants change their speaking status.
+ *
+ * It is expected that the speaking status of participant will be
+ * modified only when the current conversation is joined and call is started.
+ *
+ */
+export default class SignalingSpeakingHandler {
+
+	// Constants, properties
+	#store
+	#localMediaModel
+	#callParticipantCollection
+
+	// Methods (bounded to have access to 'this')
```

Comment:
It should be _bound_, as in past participle of _bind_, rather than _bounded_, as past participle of _bound_ :-)